linkerd/app/core/src/metrics.rs

```diff
@@ -422,7 +422,7 @@ impl FmtLabels for Class {
                 "classification=\"{}\",grpc_status=\"{}\",error=\"\"",
                 class(res.is_ok()),
                 match res {
-                    Ok(code) | Err(code) => code,
+                    Ok(code) | Err(code) => Into::<i32>::into(*code),
```

Comment:
nit, not important: would it be slightly neater to write:
```suggestion
                    Ok(code) | Err(code) => <i32>::from(*code),
```